### DISCUSSION TITLE
new schema not found on `auto complate` / `IntelliSense` ?

### DISCUSSION THREAD

**Discussion by d-strygwyr** [chakra-ui/chakra-ui#4909] [Q&A]
ex: I have created color schema `primary` and `secondary`, if I look from the log I got that schema object.

![color](https://user-images.githubusercontent.com/49369600/138567494-5d0b02b0-6683-4cd6-8625-711e0628bca9.png)

but my new schema not found on `auto complate` / `IntelliSense` 

**Try Find `Primary`**
![not defined](https://user-images.githubusercontent.com/49369600/138567648-33467b65-17da-45e4-ac4d-16236871f458.png)

**Try Find `Secondary`**
![image](https://user-images.githubusercontent.com/49369600/138567677-08048f52-8a45-4ff2-a0be-75ab242c58e6.png)

**Comment by no-yan** [comment]
Suggests rely on typing of theme, but by default they rely on uncustomized one. 
So you need to generate your own typing if you want.
See: https://chakra-ui.com/docs/theming/advanced#theme-typings

**Reply by d-strygwyr** [reply]
btw what mean `tokens`?

any example repo ?

**Reply by d-strygwyr** [reply]
success, but `primary` or `secondary` still not show 😢 

![image](https://user-images.githubusercontent.com/49369600/138602317-bd70d217-7428-4b23-9b93-a9cf2e5552c9.png)

**Reply by d-strygwyr** [reply]
btw this for my repo [Link](https://github.com/hafidzamr/react-redux-ts)

**Comment by d-strygwyr** [comment]
any update for this ?
